### ISSUE TITLE
Connection Error

### ISSUE THREAD

**Issue by shellylac** [ropensci/geonames#21]
when I try to run this (with the name I registered with geonames.org):

`library(geonames)
options(geonamesUsername="myusername")
countryInfo <- GNcountryInfo()`

I get an error message:

Error in url(url, open = "r") : cannot open the connection
In addition: Warning message: In url(url, open = "r") :  cannot open URL 'http://api.geonames.org/countryInfo?country=&lang=&type=JSON&username=myusername': HTTP status was '401 Unauthorized'

`packageVersion("geonames")` 
[1] ‘0.999’

I can run this code with no error:
`readLines(url("http://www.example.com/",open="r"))`

But this: 
`readLines(url("http://api.geonames.org/",open="r"))`

results in an error:
Error in url("http://api.geonames.org/", open = "r") :   cannot open the connection
In addition: Warning message: In url("http://api.geonames.org/", open = "r") :
  cannot open URL 'http://api.geonames.org/': HTTP status was '401 Unauthorized'

**Comment by barryrowlingson** [comment]
The package works fine for me with my username, and if I use an invalid username I get the error you see:

```
> options(geonamesUsername="nas98ajh4ds3")
> countryInfo <- GNcountryInfo()
Error in url(url, open = "r") : 
  cannot open the connection to 'http://api.geonames.org/countryInfo?country=&lang=&type=JSON&username=nas98ajh4ds3'
In addition: Warning message:
In url(url, open = "r") :
  cannot open URL 'http://api.geonames.org/countryInfo?country=&lang=&type=JSON&username=nas98ajh4ds3': HTTP status was '401 Unauthorized'
```

So I can only conclude that your username isn't authorized. Was it newly created? Maybe geonames is a bit slow on the updates? Hmmm

**Comment by shellylac** [comment]
I've found out the issue. I hadn't yet enabled my Geonames.org account for the free webservice. I realised this by visiting this link 
http://api.geonames.org/countryInfo?country=&lang=&type=JSON&username=myusername 
which gave the following error message
```
{
	"status": {
		"message": "user account not enabled to use the free webservice. Please enable it on your account page: https://www.geonames.org/manageaccount ",
		"value": 10
	}
}
```
I visited this site https://www.geonames.org/manageaccount and enabled the free webservice and then the R package worked fine.
